src/llmq/utils.cpp

```diff
@@ -244,16 +244,16 @@ std::vector<std::vector<CDeterministicMNCPtr>> ComputeQuorumMembersByQuarterRota
 
     for (const size_t i : irange::range(nQuorums)) {
         for (const auto &m: previousQuarters.quarterHMinus3C[i]) {
-            quorumMembers[i].push_back(std::move(m));
+            quorumMembers[i].push_back(m);
         }
         for (const auto &m: previousQuarters.quarterHMinus2C[i]) {
-            quorumMembers[i].push_back(std::move(m));
+            quorumMembers[i].push_back(m);
         }
         for (const auto &m: previousQuarters.quarterHMinusC[i]) {
-            quorumMembers[i].push_back(std::move(m));
+            quorumMembers[i].push_back(m);
         }
         for (const auto &m: newQuarterMembers[i]) {
-            quorumMembers[i].push_back(std::move(m));
+            quorumMembers[i].push_back(m);
```

Comment:
Why no need move here and why does it has been introduced?

Reply:
m is a const&; a const& cannot be cast to a && so this std::move does nothing but confuse

Reply:
I think this code would've been introduced by @ogabrielides; not sure why he used std::move :)

Reply:
Maybe better to remove const then?

Reply:
not quite that simple but please see https://github.com/dashpay/dash/pull/5918

Reply:
should drop these changes from this PR then

Reply:
yup; done was just waiting to see if the other would gain traction.